generators/app/templates/templates/component/index.test.tsx.hbs

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import {{name}} from './index';
+
+test('renders correctly without the optionalProp', () => {
+  const { container } = render(<{{name}} someProp={1} />);
+  expect(container.firstChild).toMatchSnapshot();
+});
+
+test('renders correctly with the optionalProp', () => {
+  const { container } = render(<{{name}} someProp={1} someOptionalProp={2} />);
+  expect(container.firstChild).toMatchSnapshot();
+});
+
+// You can test simple components with snapshots but if
```

Comment:
Same as @damfinkel and the comment could be `You can use snapshot tests for simple components, but if the component has any interaction or conditional logic you should add tests for those.`